pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -1,196 +1,205 @@
-{ lib, stdenv, fetchurl, pkgconfig, gtk2, pango, perl, python2, python3, nodejs
-, libIDL, libjpeg, zlib, dbus, dbus-glib, bzip2, xorg
-, freetype, fontconfig, file, nspr, nss, libnotify
-, yasm, libGLU, libGL, sqlite, zip, unzip
-, libevent, libstartup_notification
-, icu, libpng, jemalloc
-, autoconf213, which, m4, fetchpatch
-, writeScript, xidel, common-updater-scripts, coreutils, gnused, gnugrep, curl
-, runtimeShell
-, cargo, rustc, rust-cbindgen, llvmPackages, nasm
-, enableGTK3 ? false, gtk3, gnome3, wrapGAppsHook, makeWrapper
-, enableCalendar ? true
+{ lib, fetchurl, makeDesktopItem, makeWrapper, runtimeShell, stdenv
+, autoconf213, dbus, m4, libIDL, nspr, pkgconfig
+, bzip2, file, gnused, unzip, which, zip
+, cargo, nasm, nodejs, rustc, perl, python2, python3, yasm
+, dbus-glib, glib, jemalloc, libevent, nss, sqlite, zlib
+, fontconfig, freetype, icu, pango
+, libGL, libGLU, xorg , libjpeg, libpng
+, libnotify, libstartup_notification
+, llvmPackages, rust-cbindgen
+, common-updater-scripts, coreutils, curl, gnugrep, writeScript, xidel
 , debugBuild ? false
+
+, gtk3Support ? true, gtk2, gtk3, wrapGAppsHook
+, waylandSupport ? true, libxkbcommon
```

Comment:
does building with waylandSupport cause it to break under x11?

Reply:
Nope.